backend/cmd/server/main.go

```diff
@@ -69,6 +69,10 @@ func NewTemporalClient(host, namespace string) (client.Client, error) {
 func main() {
 	_ = godotenv.Load()
 
+	if os.Getenv("DEBUG_AUTH_EMAIL") != "" {
+		fmt.Printf("DEBUG_AUTH_EMAIL: %s\n", os.Getenv("DEBUG_AUTH_EMAIL"))
```

Comment:
Is this supposed to be here in prod?

Reply:
Yes, It's just a notice that you can validate that the thing is actually set when running it.